README.md

```diff
@@ -1,9 +1,9 @@
 About loky
 ==========
 
-Home: https://github.com/joblib/loky
+Home: https://github.com/tomMoral/loky
```

Comment:
There seems to be an issue here as the README is updated in the wrong direciton

Reply:
hum... probably a metadata file is wrong somewhere.